crates/control/src/behavior/calibrate.rs

```diff
@@ -1,12 +1,24 @@
 use types::{
-    motion_command::{HeadMotion::Unstiff, MotionCommand},
+    calibration::CalibrationCommand,
+    motion_command::{HeadMotion, ImageRegion, MotionCommand},
     primary_state::PrimaryState,
     world_state::WorldState,
 };
 
 pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
-    match world_state.robot.primary_state {
-        PrimaryState::Calibration => Some(MotionCommand::Stand { head: Unstiff }),
-        _ => None,
+    if PrimaryState::Calibration != world_state.robot.primary_state {
+        return None;
     }
+
+    let Some(CalibrationCommand::LookAt { target, camera, .. }) = world_state.calibration_command
+    else {
+        return None;
```

Comment:
This currently causes a panic because there is no other action that feels responsible to produce a motion command in calibration mode.

Reply:
Oh I know what happened, there were 3 actions in the old version,

* If not calibration -> nothing 
* If calibration and look at -> look at head motion 
* If calibration but not look at -> head motion center

So the last condition has to be split to ternary expression style with restructuring to handle the last two cases.

Reply:
In the third case, the robot would already start to look away from the target while vision tries to capture measurements, wouldn't it?

Reply:
Hmm that's also right.
I'll re check the older working state and make the nessesary changes. Unstiff or no HeadMotion might be better.

Reply:
Neither of those sound like a good idea, you want neither head nor body to move during the capture.
That's the whole point of waiting in the LookAt state, isn't it?

Reply:
Oh damn, I had forgotten it! One option is to go with "Unstiff" -> not reliable, head may or may not move.

Or Combine look at and capture
```rust
enum State{

LookAtAndCapture {
  dispatch_time: ...,
  target: point![1.0, 0.0],
  camera: CameraPostion:Top,
  trigger: true // Controller flips this On when the timeout is reached. used by the vision cycler to commence capturing
}
```

Reply:
*or*, I tried this, the command is now called `CalibrationCaptureCommand`, IDK if this is the best way to do it but simplify the consumer's logic.

As for the internal state, I kept the seperate states as I felt it was more readable in that context.